packages/protocol/contracts/locking/TimeLockManager.sol

```diff
@@ -12,7 +12,10 @@ import "../locking/IStakingLocking.sol";
  * Allows to set a time interval, either in blocks or seconds, during which the funds are locked.
  * Outside that window the owner can unlock them.
  */
+// Note: in terms of auditing this, is this something that is more like an example (and can be moved
+// to be an example that is not directly audited with Staking's core components)?
```

Comment:
Good question. We are not using, but it seems like a useful example, and probably the first one that popped up when we started working in Staking long time ago.